app/scripts/controllers/otherResources.js

```diff
@@ -70,6 +70,7 @@ angular.module('openshiftConsole')
       if (!selected) {
         return;
       }
+      $scope.selectedResource = APIService.kindToResource(selected.kind);
```

Comment:
@jhadvig so you are seeing it being returned as "horizontalpodautoscalers" not "extensions/horizontalpodautoscalers" ?   you can have the same "kind" in more than one group, i'd expect to get cases where the kind from one group has a different set of allowed actions than the kind in another group. Other than some special cases, the same kind in different groups actually are different resources.

cc @deads2k